src/main/java/data/UserComment.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+ 
+/** A user's comment, with corresponding user info. */
+public class UserComment {
```

Comment:
For integration with other features: UserComment should contain the id of the user that posted it, and the id of the recipe that it's posted on.

Reply:
got it - will change after that code is merged in/pulled so i can make sure it works w it